src/lib.rs

```diff
@@ -224,30 +220,39 @@ pub fn server_login_start(
     )
     .map_err(from_protocol_error("start serverLogin"))?;
 
-    let credential_response = BASE64.encode(server_login_start_result.message.serialize());
-    let server_login = BASE64.encode(server_login_start_result.state.serialize());
+    let login_response = BASE64.encode(server_login_start_result.message.serialize());
```

Comment:
nit pick: can we rename also the internals `server_login_start_result` -> `start_server_login_result`